src/platforms/python/common/troubleshooting.mdx

```diff
@@ -114,3 +115,58 @@ be fixed from within the SDK.
 This [issue has been fixed with gevent
 20.5](https://github.com/gevent/gevent/issues/1407) but continues to be one for
 eventlet.
+
+## Django Channels
+
+A Django application using Channels 2.0 will be correctly instrumented under Python 3.7. For older versions of Python, install `aiocontextvars` from PyPI or your application will not start.
+
+If you experience memory leaks in your channels' consumers while using the SDK, you need to wrap your entire application in [Sentry's ASGI middleware](/guides/asgi/).Unfortunately the SDK is not able to do so by itself, as [Channels is missing some hooks for instrumentation](https://github.com/django/channels/issues/1348).
+
+## Matching Behavior in the Django integration
```

Comment:
everything starting frmo here to the bottom of the page should not be visible outside of Django and is IMO misplaced in Troubleshooting

Reply:
I'll move some of this into the verification step, and try the PlatformSection toggle for what remains.